.github/ISSUE_TEMPLATE/---bugs.md

```diff
@@ -0,0 +1,29 @@
+---
```

Comment:
There're issues with the filenames. Could you rename the file to `bugs.md`

Reply:
all done sir